app/models/plant_line.rb

```diff
@@ -45,6 +45,9 @@ def self.grid_data(filter)
       .pluck(*columns)
   end
 
+  def self.genetic_statuses
+    order('genetic_status').pluck('DISTINCT genetic_status').reject(&:blank?).reject { |s| ['none', 'not applicable', 'unspecified'].include?(s) }
```

Comment:
Well, I'm not a 80-char zealot ;) but this is just a bit too much. Try to wrap it.